### ISSUE TITLE
Zoom out: Add an Editor setting to disable Zoom out mode

### ISSUE THREAD

**Issue by ndiego** [WordPress/gutenberg#63584]
The experimental [Zoom out mode](https://github.com/WordPress/gutenberg/issues/50739) is very exciting and should make composing sites with patterns much easier for the vast majority of WordPress users. 

That said, there are going to be scenarios where the mode is not what the user/developer wants. Perhaps they only use a few patterns, and they are not designed as sections. Either way, when the mode is officially released, it should come with an Editor setting that allows developers to easily disable it if needed in their environment using existing filters like `block_editor_settings_all`.

**Comment by maddisondesigns** [comment]
Agreed! We need an option to disable this annoying functionality. Shrinking the layout doesn't automagically make the page easier to edit. It's also supposed to 'Zoom Out' when you switch back to the Blocks tab, but it's unreliable and that doesn't always happen. This option needs to be added before 6.7 is rolled out.

**Comment by xenio** [comment]
Is there an option to disable Zoom out in WP 6.7.1? I can't find it, it is very annoying feature. :(

**Comment by Adi77** [comment]
Please add an option to disable the zoom out function.

**Comment by carolinan** [comment]
There is not much detail, but **my interpretation** of comments on the linked PR, is that `block_editor_settings_all` should no longer be used. Without having the background information about why, my opinion is that implementing it as a Rest API filter is more difficult both for the implementor (hi) and the developer who wants to manage which features that are available.
I need more experienced contributors to explain and motivate why the Rest API would be the preferred choice.
The developer experience needs to be considered too, there are now at least 4 different ways to disable different features using PHP and its not ideal. (I'm not even counting JS)


Now for the setting or filter, how granular is it expected to be?
- On/off
- Per post type?
- template post type only?

**Comment by maddisondesigns** [comment]
@carolinan My preference would be for an option to simply disable it completely (and also remove the button). The option should be added to the Preferences UI as well, and not force people to have to add PHP to disable it.

**Comment by turansadri** [comment]
Is there still a way to disable the Zoom out mode?

**Comment by turansadri** [comment]
Is there still a way to disable the Zoom out mode?
